src/components/TidligstMuligUttaksalder/__tests__/TidligstMuligUttaksalder.test.tsx

```diff
@@ -274,17 +275,14 @@ describe('TidligstMuligUttaksalder', () => {
         />
       )
       expect(
-        await screen.findByText(
-          'alderspensjon fra 67 år. Vil du beregne uttak før 67 år, må du gå til ',
-          {
-            exact: false,
-          }
-        )
+        await screen.findByText('Vil du beregne uttak før ', {
```

Comment:
Hvis du synes det er dumt at teksten du sjekker mot blir så kort, kan du gjøre noe sånt som dette:
```suggestion
        await screen.findByText('Vil du beregne uttak før , må du gå til', {
```
Fant det ut ved å teste her: https://testing-playground.com/

Reply:
Ahaaa, takk det var bra tips det :)